pages/documentation/simulation/sensitivity/openlf.md

```diff
@@ -76,88 +76,83 @@ We only support for the moment balance type `PROPORTIONAL_TO_GENERATION_P_MAX` a
 
-Let $$\theta^1$$ be the vector of voltage angles obtained solving the network contraints system on the pre-contingency network with right-hand side $$b^1$$.
+We call $$b^2$$ be the right-hand side vector corresponding to an increase of 1 MW at bus $$m$$ and a decrease of 1 MW at bus $$k$$. We call $$\theta^2$$ the state vector of voltage angles obtained by solving the equation system on the pre-contingency network that has as right-hand side $$b^2$$.
 
-Let $$\theta^2$$ be the vector of voltage angles obtained solving the network contraints system on the pre-contingency network with right-hand side $$b^2$$.
-Notice that $$\theta^1$$ and $$\theta^2$$ are built using the same LU decomposition of the constraints matrix $$J$$.
+Note that both $$\theta^1$$ and $$\theta^2$$ are built using the same LU decomposition of the constraints matrix $$J$$.
 
 Let $$s_{b,ij}$$ be the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ on the pre-contingency network, we recall that:
 
 $$
 s_{b,ij} = \frac{\theta^1_i-\theta^1_j}{X_{i,j}}
 $$
 
-Let $$s_{mk,ij}$$ be the sensitivity of an increase of 1 MW at bus $$m$$ plus a decrease of 1 MW at bus $$k$$, on the pre-contingency network.
-It can be easily computed through the formula:
+Let $$s_{mk,ij}$$ be the sensitivity of an increase of 1 MW at bus $$m$$ and a decrease of 1 MW at bus $$k$$, on the pre-contingency network.
+It can be easily computed through the formula, valid for all buses:
 
 $$
 s_{mk,ij} = \frac{\theta^2_i-\theta^2_j}{X_{i,j}}
 $$
 
-Then, The post contingency sensitivity $$s_{b,ij,mk}$$ satisfies:
+Then, the post-contingency sensitivity $$s_{b,ij,mk}$$ satisfies:
 
 $$
 s_{b,ij,mk} = s_{b,ij} + \frac{\theta^1_m-\theta^1_k}{X_{m,k} - (\theta^2_m-\theta^2_k)}s_{mk,ij}
 $$
 
-#### N-k case
-For the N-k case, the incident causes several outages on the network.
-As for the N-1 case, an outage is the loss of a network line (this includes the special case of the loss of a transformer).
- 
-Let $$s_{b,ij,I}$$ be the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ when the incident $$I$$ occurs.
-Incident $$I$$ corresponds to the loss of the $$k$$ branches $$(m_1,n_1), \cdots, (m_k,n_k)$$.
-We want to compute this sensitivity.
+#### Loss of more than one branch
 
-Let $$b^1$$ be the right-hand side vector corresponding with an increase of 1 MW at bus $$b$$.
+Sometimes, an event in the network causes the loss of several buses and branches. Connected to these lost buses, we can have generators or loads.
+ 
+Let's introduce $$s_{b,ij,E}$$ the sensitivity of an increase of 1 MW at bus $$b$$ on branch $$(i,j)$$ when the event $$E$$ occurs. The event $$E$$ corresponds to the loss of the $$k$$ branches indexed by $$(m_1,k_1), \cdots, (m_n,k_n)$$. We want to compute this sensitivity.
```

Comment:
Here I think that we need to write "the loss of the $$n$$ branches" instead of $$k$$ branches.